src/controller/data_model/controller-clusters.matter

```diff
@@ -9134,6 +9134,56 @@ provisional cluster ContentAppObserver = 1296 {
   command ContentAppMessage(ContentAppMessageRequest): ContentAppMessageResponse = 0;
 }
 
+/** Supports the ability for clients to request the commissioning of themselves or other nodes onto a fabric which the cluster server can commission onto. */
+cluster CommissionerControl = 1873 {
+  revision 1; // NOTE: Default/not specifically set
+
+  bitmap SupportedDeviceCategoryBitmap : bitmap32 {
+    kFabricSynchronization = 0x1;
+  }
+
+  info event CommissioningRequestResult = 0 {
+    int64u requestId = 0;
+    node_id clientNodeId = 1;
+    enum8 statusCode = 2;
+  }
+
+  readonly attribute SupportedDeviceCategoryBitmap supportedDeviceCategories = 0;
+  readonly attribute command_id generatedCommandList[] = 65528;
+  readonly attribute command_id acceptedCommandList[] = 65529;
+  readonly attribute event_id eventList[] = 65530;
+  readonly attribute attrib_id attributeList[] = 65531;
+  readonly attribute bitmap32 featureMap = 65532;
+  readonly attribute int16u clusterRevision = 65533;
+
+  request struct RequestCommissioningApprovalRequest {
+    int64u requestId = 0;
+    vendor_id vendorId = 1;
+    int16u productId = 2;
+    optional char_string label = 3;
```

Comment:
there is a constraint of max 64 chars on the label. Would it be worth encoding here (in case other tools make use of it)?

Reply:
Already enforced in xml 

      <arg name="Label" type="char_string"  lenght="64" optional="true"/>     
      
This constraint is not auto-reflected in .matter file.

Reply:
maybe typo of `length` I see used in other places?

Reply:
I see in the xml it looks right. Maybe not supported for commands.